content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -399,7 +399,7 @@ rolled back.
 * Press Ctrl-C to stop the above rollout status watch. For more information on stuck rollouts,
 [read more here](#deployment-status).
 
-* You see that the number of old replicas (`nginx-deployment-1564180365` and `nginx-deployment-2035384211`) is 2, and new replicas (nginx-deployment-3066724191) is 1.
+* You see that the number of old ReplicaSets (`nginx-deployment-1564180365` and `nginx-deployment-2035384211`) is 2, and new ReplicaSets (nginx-deployment-3066724191) is 1.
 
   ```shell
   kubectl get rs
```

Comment:
**Aside**: this ought to be:
```shell
  kubectl get rs -l app=nginx
```

Reply:
@sftim I agree with you that it would be better to have the selector specified.

However, after revisiting the document, it would mean to change around 10 places across the whole document. I am not sure if the change would fit in this PR.

Reply:
Yep, if changing it, a separate PR is better (IMO).